src/app/tasks/checker-inbox-and-tasks-tabs/loan-approval/loan-approval.component.ts

```
@@ -204,7 +205,7 @@ export class LoanApprovalComponent {
       response.forEach((responseEle: any) => {
         if ((responseEle.statusCode = '200')) {
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)